apps/documentation/src/routes/documentation/installation.mdx

```diff
@@ -13,25 +13,25 @@
 
 ## Requirements
 
-`Tuono` is a development environment built in rust and typescript that outputs a website written with both the
-languages. Since that you need the following tools installed on your computer to work with it:
+`Tuono` is a development environment built in rust and typescript that outputs a website written in both the
+languages. Since then, you need the following tools installed on your computer to work with it:
 
-- `Rust` - The rust programming language toolchain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
+- `Rust` - The Rust programming language tool chain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Did you mean “LanguageTool” (product name)? (LANGUAGE_TOOL[2])
Suggestions: `LanguageTool`
URL: https://en.wikipedia.org/wiki/LanguageTool 
Rule: https://community.languagetool.org/rule/show/LANGUAGE_TOOL?lang=en-US&subId=2
Category: TYPOS
<!-- __reviewdog__:Cg80MWFlZDllMTI2MjdmNTQSDExhbmd1YWdlVG9vbA== -->